docs/specs/api.md

```diff
@@ -0,0 +1,56 @@
+# API
+
+Here are the endpoints that are not part of OIDC, but essential to provide some features of Authgear.
+
+## /api/identity/biometric/remove
+
+This endpoint is for removing Biometric identity.
+The user still can remove biometric identity in the settings page.
```

Comment:
Can we put a note in the spec on where the Biometric identities should be shown and by default off?